cmd/thor/flags.go

```diff
@@ -142,4 +142,10 @@ var (
 		Value: 16,
 		Usage: "set tx limit per account in pool",
 	}
+
+	allowedPeersOnlyFlag = cli.StringSliceFlag{
```

Comment:
Should this flag be hidden? If not, I would be inclined to remove the examples, as they may clog the `help` command.

Also I think `allowed-peers` is enough, it doesn't need the `-only` IMO